### PR TITLE
fix word errors

### PR DESCRIPTION
**Pull request by zhuyuanxiang**
*Description of changes:*

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

File: chapter_recurrent-neural-networks/language-models-and-dataset.md [hunk 1]
```diff
@@ -181,7 +181,7 @@ which is equivalent to
 $$\log n_i = -\alpha \log i + c,$$
 
 where $\alpha$ is the exponent that characterizes the distribution and $c$ is a constant.
-This should already give us pause if we want to model words by count statistics and smoothing.
+This should already give us pause if we want to model words by counting statistics and smoothing.
```
Comment (astonzhang) [inline chapter_recurrent-neural-networks/language-models-and-dataset.md:184]: count is a noun here

**Comment by mli** [comment]
Job d2l-en/PR-1811/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1811/

File: chapter_recurrent-neural-networks/language-models-and-dataset.md (hunk 1 above)
Comment (zhuyuanxiang) [reply chapter_recurrent-neural-networks/language-models-and-dataset.md:184]: I found that couting statistics is a phrase, but count statistics is not.

Comment (astonzhang) [reply chapter_recurrent-neural-networks/language-models-and-dataset.md:184]: https://pubs.rsna.org/doi/pdf/10.1148/radiographics.19.3.g99ma33765

Comment (astonzhang) [reply chapter_recurrent-neural-networks/language-models-and-dataset.md:184]: You may also find "count statistics" in papers such as https://aapm.onlinelibrary.wiley.com/doi/abs/10.1002/mp.13063 :)

Comment (zhuyuanxiang) [reply chapter_recurrent-neural-networks/language-models-and-dataset.md:184]: I found that couting statistics had more usage than count statistics. Maybe count means the total number counted, but counting means the act of counting.

Comment (astonzhang) [reply chapter_recurrent-neural-networks/language-models-and-dataset.md:184]: Screen Shot 2021-06-25 at 8 21 37 PM

I feel that "count" serves this context better

Comment (zhuyuanxiang) [reply chapter_recurrent-neural-networks/language-models-and-dataset.md:184]: I could found that, in the above sentences,  "count" is the noun and used in the end of the phrase. The meaning of "count" is the total number counted. But "counting" in "counting statistics" maybe used for adjective, and it's meaning is the act of counting. So I feel that "counting" maybe the best choice for embellishing "statistics".

Comment (astonzhang) [reply chapter_recurrent-neural-networks/language-models-and-dataset.md:184]: Both nouns and adjectives are common modifiers. Nonetheless, I am leaning towards "counting statistics" after reading this paper: https://arxiv.org/pdf/cond-mat/9607137.pdf

Comment (zhuyuanxiang) [reply chapter_recurrent-neural-networks/language-models-and-dataset.md:184]: OK.
Glad to communicate with u.

Comment (zackchase) [reply chapter_recurrent-neural-networks/language-models-and-dataset.md:184]: The whole sentence here is ungrammatical. I'll revise.

**Pull request merged**